tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Do we want to links to authors? i.e.</p><ul><li>Author: <a href="https://github.com/jdbcode" target="_blank">jdbcode</a></li></ul><p> </p><p>Suggestion to make use of articles consistent:</p><ul><li>Formatting a table in Earth Engine</li><li>Transferring <u>an</u> Earth Engine table to <u>a</u> Colab Python kernel</li><li>Converting <u>an</u> Earth Engine table to&nbsp;<u>a</u> <a href="https://pandas.pydata.org/" target="_blank">pandas</a>&nbsp;DataFrame</li></ul><p> </p><p>The following seems unnecessary, given that you are already linking to the page in the previous sentence.</p><p>"For a review of Python syntax as it relates to the Earth Engine API, please see this <a href="https://developers.google.com/earth-engine/python_install#syntax" target="_blank">guide</a>."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Re: articles - Done here and throughout.</p><p> </p><p>Re: remove redundant syntax link - Done.</p>

Reply:
<p>"Earth an Engine table" -&gt; "an Earth Engine table"</p>